qiskit/transpiler/passes/utils/check_map.py

```diff
@@ -45,16 +47,35 @@ def __init__(self, coupling_map, property_set_field=None):
         else:
```

Comment:
Something I've noticed but can't comment on directly: 
The line
```
store the result of the check. If not default the result is stored in
``` 
is misleading. It should actually be: 
```
store the result of the check. If not provided the result is stored in
```

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/14925